app/src/main/java/com/parishod/watomatic/fragment/MainFragment.java

```diff
@@ -273,6 +275,18 @@ public void onClick(View v) {
         }
     }
 
+    //REF: https://stackoverflow.com/questions/37539949/detect-if-an-app-is-installed-from-play-store
```

Comment:
Should be moved to a utility class if possible.
Possibly in this PR or later if not.

Reply:
we have few things to move to utilities we'll do all of them in separate issue